test/e2e/setup.go

```diff
@@ -0,0 +1,78 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/client-go/tools/clientcmd/api/v1"
+
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/redhatopenshift"
+	utillog "github.com/Azure/ARO-RP/pkg/util/log"
+	"github.com/Azure/ARO-RP/test/clients/openshift"
+)
+
+type ClientSet struct {
+	log *logrus.Entry
+
+	openshiftclusters redhatopenshift.OpenShiftClustersClient
+	openshiftclient   *openshift.Client
+}
+
+var (
+	Clients *ClientSet
```

Comment:
+1 on making this global; I feel tempted to do the same with `Log *logrus.Entry` - doesn't have to be now.